src/bp-templates/bp-nouveau/buddypress-functions.php

```diff
@@ -122,7 +122,7 @@ function() {
 			);
 
 			// When BP Classic is activated, regular themes need this filter. 
-			if ( function_exists( 'bp_classic' ) ) {
+			if ( function_exists( 'bp_classic' ) || is_plugin_active( 'bp-classic/bp-classic.php' ) || is_plugin_active( 'buddyclassic/bp-classic.php' ) ) {
```

Comment:
Is `buddyclassic` a real plugin?

Reply:
@renatonascalves Thanks for reviewing this PR. I have added this just for the case when user will rename folder to `buddyclassic` from `bp-classic`

Reply:
@renatonascalves Please let me know if I need to made any other changes in this.

Reply:
I'd remove the buddyclassic part, actually. It is not a _real_ plugin.